### PR TITLE
MAUT-8716 added cortex file to MauticSlooceTransportBundle

### PR DESCRIPTION
**Pull request by aarohiprasad**
https://backlog.acquia.com/browse/MAUT-8716

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -34,6 +34,7 @@ pipeline {
               sh("composer config --global github-oauth.github.com ${composertoken}")
             }
             sh '''
+              git config --global --add safe.directory '*'
```
Comment (avikarshasaha) [inline Jenkinsfile:37]: Why are we including a safe directory? Are you experiencing pipeline errors?

Comment (fedys) [reply Jenkinsfile:37]: @avikarshasaha yes, there was a pipeline issue. See https://core.cloudbees.ais.acquia.io/mautic-jenkins/job/mc-cs-plugins/job/mc-cs-plugin-slooce/job/PR-22/1/console

**Review by fedys — APPROVED** [review]
Thanks @aarohiprasad!

File: Jenkinsfile (hunk 1 above)
Comment (aarohiprasad) [reply Jenkinsfile:37]: Yes, I was facing[ this](https://acquia.slack.com/archives/CJHAV5Y91/p1675068440463909) exact error. I followed the solution suggested by @himanshu8acquia

**Review by avikarshasaha — APPROVED** [review]

**Comment by tejasync** [comment]
Commenting to watch this PR.

**Pull request merged**